content/offices/brisbane.mdx

```diff
@@ -57,7 +57,8 @@ aboutUs: >
   Server or Visual Studio Online, and Microsoft Azure.
 map: /images/offices/brisbane/SSW-brisbane-map-wide.jpg
 directions: >
-  #### [Level 1, 471 Adelaide St, Brisbane, QLD 4000,
+  ####  [<CustomImage altText="map-icon" height={16} width={16}
```

Comment:
given that all this is repeated for each use of the directions prop - I think this should be part of the page template instead of repeated for each city

Reply:
I tried to add the SVG image to the page template, but it doesn't work. This is because we are using TinaMackdown to render the entire directions section and cannot insert the img tag inside the h4 tag.